i18n/translations/fr/common.json

```diff
@@ -38,6 +38,7 @@
     "heading": "Veuillez corriger les erreurs sur la page.",
     "required": "Compléter le champ obligatoire afin de continuer.",
     "alphanumeric": "Entrer uniquement des lettres et des chiffres.",
+    "duplicateEmail": "[FR] This email was already added.",
```

Comment:
@anikbrazeau 

Fresh string here --- 

Using Dave's tag input but for emails.